src/cunumeric/index/wrap.cu

```diff
@@ -32,40 +32,44 @@ __global__ static void __launch_bounds__(THREADS_PER_BLOCK, MIN_CTAS_PER_SM)
               const Point<1> out_lo,
               const Pitches<DIM - 1> pitches_in,
               const Point<DIM> in_lo,
-              const size_t in_volume)
+              const size_t in_volume,
+              IND indices)
```

Comment:
Can we pass the `IND` argument with `const`, just to match the other cases? I.e. `const IND indices` for kernels, and `const IND& indices` on host-side functions (in this and other files?)

It might just be me, but anytime I see a non-const `Foo&` argument, that signals to me that this is an "output" argument (which is not the case here AFAICT).

Reply:
sure, thanks